test/datasets_utils.py

```diff
@@ -357,13 +357,31 @@ def _verify_required_public_class_attributes(cls):
 
     @classmethod
     def _populate_private_class_attributes(cls):
-        argspec = inspect.getfullargspec(cls.DATASET_CLASS.__init__)
```

Comment:
This fix is needed to be able to detect a default config for datasets that use `**kwargs` in their constructor. For example

https://github.com/pytorch/vision/blob/bb2805a669b197582b796f0d86ef8932e1a4396a/torchvision/datasets/mnist.py#L289

https://github.com/pytorch/vision/blob/bb2805a669b197582b796f0d86ef8932e1a4396a/torchvision/datasets/mnist.py#L392-L394